lib/src/module.dart

```diff
@@ -142,6 +142,8 @@ abstract class Module {
     } else if (reserveDefinitionName &&
         !Sanitizer.isSanitary(definitionName!)) {
       throw InvalidReservedNameException();
+    } else if (reserveDefinitionName && definitionName!.isEmpty) {
+      throw NullReservedNameException();
```

Comment:
I think this should be an exception for invalid, not null?  Also, can't this not get hit because the sanitizer will always catch it?

Reply:
Okay, I swap the priority now and create a new exception on empty string. Feel like better to separate the exception message to avoid confusion.